jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -86,6 +84,10 @@
       <unit id="org.apache.jasper.glassfish" version="2.2.2.v201205150955"/>
       <!-- <unit id="ch.qos.logback.slf4j" version="1.0.7.v20121108-1250"/> -->
       <unit id="org.slf4j.api" version="1.7.2.v20121108-1250"/>
+      
+      <!-- Testing -->
+      <unit id="org.mockito" version="1.9.5.v201311280930"/>
```

Comment:
The patched Mockito has never been included (it was commented). So it's not moved away.